dispatch/dispatch.go

```diff
@@ -81,6 +81,7 @@ type AlertBlock struct {
 type APIAlert struct {
 	*model.Alert
 	Status types.AlertStatus `json:"status"`
+	ID     string            `json:"id"`
```

Comment:
I'm not a fan of hash since that's naming it after its implementation. I'm fine with `id` for now since the fingerprint is how we're identifying the alerts.